docs/source/disable-warnings.py

```diff
@@ -1,3 +1,6 @@
+# flake8: noqa 401 imported to force console mode for tqdm in jupyter notebooks
+from tqdm.auto import tqdm
```

Comment:
Also added this import to get rid of this error `Using `tqdm.autonotebook.tqdm` in notebook mode. Use `tqdm.tqdm` instead to force console mode (e.g. in jupyter console)` that appears in all the notebooks. Let me know if you think we should add it to a different file or anything